### ISSUE TITLE
This has bugged me for the last 5 years. Please allow global configs

### ISSUE THREAD

**Issue by Pangamma** [LogBlock/LogBlock#744]
The one thing that really bugs me about logblock is I have to re-configure it for every single world I have. I really want to be able to re-use the same exact settings for every world. Please add a global configuration?

**Comment by KleinCrafter** [comment]
Yes, this is bugging me too, so I try to implement it without breaking too much..

The Plugin directory can be changed by startup parameter, for simplicity, I use `plugins/LogBlock/` in examples.
The World container directory can be changed by startup parameter, for simplicity, I use `/worlds/` is examples.

My approach is inspired by LuckPerms and paper, for the layers of configurations.
- Environment variables
- System Properties
- world spezific configuration
  - Insite the world directory (`/worlds/world/logblock.yml`)
  - Insite the plugin directory (`plugins/LogBlock/worlds/world.yml`)
- "default" World configuration (`plugins/LogBlock/world-default.yml`)

The Environment variables and System Properties are relevant for containerized servers, the reason I want to implement this ;)

Existing world configurations would be moved to `plugins/LogBlock/worlds/*.yml`

**Comment by KleinCrafter** [comment]
I think a dimension spezific world configuration layer is an good addition:
```
/worlds/world/logblock.yml
plugins/LogBlock/worlds/world.yml
plugins/LogBlock/world-dimension.yml
plugins/LogBlock/world-default.yml
```

**Comment by Brokkonaut** [comment]
Looks like a good idea, but I don't think we should have the dimension layer, mostly because the distinction between worlds/dimensions is a pure (historical) bukkit thing, and with the new data driven dimension generator the three main dimensions are nothing special, any new dimension can be added and configured with datapacks.

But maybe we could have some kind of templates or inheritance for the world configs to avoid duplications there
